walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,126 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.HashSet;
+import java.util.Set;
+
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    private static int start;
+    private static int end;
+    private static TimeRange event_time;
+    private static TimeRange event_1;
+    private static TimeRange event_2;
+
+  Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+	long current_duration = request.getDuration();
+    Collection<String> request_attendees = request.getAttendees();
+    String only_event_attendee;
+    Collection<TimeRange> Event_Times = new ArrayList<>();
+    List<Integer> Event_Start = new ArrayList<Integer>();
+    List<Integer> Event_End = new ArrayList<Integer>();
+    
+
+    if (events.isEmpty()) {
+        if (current_duration > (24 * 60)) { // noOptionsForTooLongOfARequest
+            return Arrays.asList();
+        }
+
+        else if (request_attendees.isEmpty()) { // noOptionsForTooLongOfARequest
```

Comment:
Please group the lines like this:

if (c1) {
   // ....
} else if (c2) {
  // .....
} else {
 //....
}